Procfile.dev.sc

```diff
@@ -0,0 +1,2 @@
+rails: ./bin/rails server -p 5000 -b 0.0.0.0
```

Comment:
had to make a special procfile to get the binding right on rails. If there's a way to parse environment variables internally in the procfile we could do that here.